src/i18n/pt-BR/index.js

```diff
@@ -6,85 +6,85 @@ export default {
     message: {
       ...potato,
       help: {
-        label: 'Summon Wizard'
+        label: 'Sumonar mago'
       },
       print: { ...print },
       tester: { ...tester },
       keyboard: {
-        label: 'keyboard'
+        label: 'telcado'
       },
       layout: {
         label: 'layout'
       },
       keymapName: {
-        label: 'Keymap Name',
-        placeholder: 'custom keymap name'
+        label: 'Nome da keymap',
+        placeholder: 'nome customizavel da keymap'
       },
       loadDefault: {
-        label: 'Load Default',
-        title: 'Load default keymap from QMK Firmware'
+        label: 'Carregar padrão',
+        title: 'Carregar keymap padrão do QMK Firmware'
       },
       compile: {
-        label: 'Compile',
-        title: 'Compile Keymap'
+        label: 'Compilar',
+        title: 'Compilar Keymap'
       },
       downloadKeymap: {
-        title: 'Download keymap.c only',
-        label: 'keymap only'
+        title: 'Baixar somente keymap.c',
+        label: 'soment keymap'
       },
       downloadSource: {
         title: 'Download QMK Firmware code',
         label: 'Full Source'
       },
       downloadJSON: {
-        title: 'Export QMK Keymap JSON file',
+        title: 'Exportar arquivo JSON QMK Keymap',
         label: 'Keymap.JSON'
       },
       importJSON: {
-        title: 'Import QMK Keymap JSON file'
+        title: 'Importar arquivo JSON QMK Keymap'
       },
       importUrlJSON: {
-        title: 'Import QMK Keymap JSON from URL'
+        title: 'Importar JSON da URL QMK Keymap JSON'
       },
       printKeymap: {
-        title: 'Print Keymap Layers',
-        label: 'Print Keymap'
+        title: 'Printar camadas de Keymap',
+        label: 'Printar Keymap'
       },
       testKeys: {
-        title: 'Test Keyboard Input',
-        label: 'Test Keyboard'
+        title: 'Testar entrada do teclado',
+        label: 'Testar teclado'
       },
       downloadFirmware: {
         label: 'Firmware',
-        title: 'Download firmware file for flashing'
+        title: 'Baixar arquivo firmware para flashing'
       },
       flashFirmware: {
         label: 'Auto-Flash',
-        title: 'Automatically Flash compiled Firmware to MCU'
+        title: 'Flash compilado automaticamente de Firmware para MCU'
       },
       flashFile: {
-        label: 'Custom-Flash',
-        title: 'Flash User Selected file to MCU'
+        label: 'Flash-Customizavel',
```

Comment:
I would use “Customizado” and not “Customizável“, which is also missing the accent.